tests/models/bert/test_tokenization_bert_tf.py

```diff
@@ -41,13 +41,8 @@ class BertTokenizationTest(unittest.TestCase):
     def setUp(self):
         super().setUp()
 
-        self.tokenizers = [
-            BertTokenizer.from_pretrained(checkpoint) for checkpoint in (TOKENIZER_CHECKPOINTS * 2)
-        ]  # repeat for when fast_bert_tokenizer=false
-        self.tf_tokenizers = [TFBertTokenizer.from_pretrained(checkpoint) for checkpoint in TOKENIZER_CHECKPOINTS] + [
-            TFBertTokenizer.from_pretrained(checkpoint, use_fast_bert_tokenizer=False)
-            for checkpoint in TOKENIZER_CHECKPOINTS
-        ]
+        self.tokenizers = [BertTokenizer.from_pretrained(checkpoint) for checkpoint in TOKENIZER_CHECKPOINTS]
+        self.tf_tokenizers = [TFBertTokenizer.from_pretrained(checkpoint) for checkpoint in TOKENIZER_CHECKPOINTS]
```

Comment:
Note to probably-Amy here: I'm not testing the old slow Bert tokenizer here because the fast one seems generally better maintained and is fully supported in TFLite now.